bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifier.scala

```diff
@@ -136,7 +136,7 @@ trait BagVerifier[BagContext <: BagVerifyContext[BagPrefix], BagLocation <: Loca
       _ <- verifyFetchPrefixes(
         fetch = bag.fetch,
         root = S3ObjectLocationPrefix(
-          bucket = primaryBucket,
+          bucket = namespace,
```

Comment:
The use of "bucket" rather than "namespace" is deliberate here – the primary copy of a bag is always in an S3 bucket, not an Azure container or anywhere else.